tests/Google/Ads/GoogleAds/V1/Util/PartialFailuresTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Ads\GoogleAds\V1\Util;
+
+use \Google\Protobuf\Internal\Message;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Unit tests for `PartialFailures`.
+ *
+ * @see PartialFailures
+ * @small
+ */
+class PartialFailuresTest extends TestCase
+{
+    public function testIsPartialFailure()
+    {
+        $messageMock = $this
```

Comment:
Instead of mocking many thing, can you research to see if we can just instantiate a Message?
We should avoid doing "over-mocking", when possible.

Reply:
Agreed, I replaced mocks with actual object instances in most cases. I left the mocks where creating an object with the expected behavior was not immediate, such as driving `any->unpack` or message serialization.